ebs/error_budget.py

```diff
@@ -417,6 +410,407 @@ def run_etc(self, wfe, wfsc_factor, sensitivity
             self.output_to_json(output_filename_prefix+'.json')
 
 
+class ErrorBudgetMcmc(object):
```

Comment:
this seems like a lot of repeat code form the ErrorBudget class. To me this is indicative that we should probably have the ErrorBudget class be a superclass and and have the ErrorBudgetMcmc class inherit from that

Reply:
ErrorBudgetMcmc does inherit code from ErrorBudget for computing ppFact and delta_contrast, but other methods are either new or significantly different.  I would suggest merging the two classes using ErrorBudgetMcmc as the base and incorporating necessary methods from ErrorBudget.

Reply:
You're right - I was looking mainly at the first part of the function/the attributes and spoke too soon. I would need to think a little more about the best way to consolidate them so maybe we can table that for a later discussion/PR since it doesn't impact the functionality